pacmod_msgs/msg/PacmodCmd.msg

```diff
@@ -10,6 +10,10 @@ uint16 TURN_RIGHT = 0
 uint16 TURN_NONE = 1
 uint16 TURN_LEFT = 2
 
+uint16 REAR_PASS_DOOR_NEUTRAL = 0
+uint16 REAR_PASS_DOOR_OPEN = 1
+uint16 REAR_PASS_DOOR_CLOSE = 2
+
```

Comment:
`PacmodCmd` only applies to PACMod2 boards. Since this platform will only support PACMod3, you can remove this change.